modules/swagger-codegen/src/main/resources/csharp/Logger.mustache

```diff
@@ -0,0 +1,55 @@
+{{>partial_header}}
+using System;
+
+namespace {{packageName}}.Client {
+    public partial class Logger {
+        public Logger(string loggerName) {
+            Initialize(loggerName);
+        }
+
+        partial void Initialize(string loggerName);
+
+        public void Trace(string message) {
+            TracePartial(message);
+        }
+
+        partial void TracePartial(string message);
```

Comment:
Definitely, it was something that I tried to workaround :)  I will try to find why I did so and let you know.